alibi_detect/utils/tests/test_prediction.py

```diff
@@ -3,7 +3,7 @@
 import tensorflow as tf
 from tensorflow.keras.layers import Dense, InputLayer
 from alibi_detect.utils.prediction import predict_batch
-from alibi_detect.models.autoencoder import AE
+from alibi_detect.models.tensorflow.autoencoder import AE
```

Comment:
I believe this file is a duplicate of `tensorflow/test_prediction.py` and can be deleted.

Reply:
It's not a duplication, test another prediction function used within the outlier detectors. I want to address this in a follow up PR to unify both.